packages/lib/src/components/SecuredFields/SecuredFields.tsx

```diff
@@ -40,6 +41,10 @@ export class SecuredFieldsElement extends UIElement {
         };
     }
 
+    protected submitAnalytics(analyticsObj: SendAnalyticsObject) {
```

Comment:
This has come up before...
For now I think this is the exception (passing all the props) rather than the rule - so I've been explicitly extending the `submitAnalytics` function so we can keep a track of where we are doing it (basically in Wallet & Card related comps.
If it turns out that we end up doing this more often (so it becomes the rule) - then we can make the refactor you suggest